src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -2207,13 +2207,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
                     Return DeclarationKind.Namespace
                 Case SyntaxKind.ImportsStatement
                     Return DeclarationKind.NamespaceImport
-                Case SyntaxKind.ClassBlock
+                Case SyntaxKind.ClassBlock, SyntaxKind.ClassStatement
                     Return DeclarationKind.Class
```

Comment:
@AlekseyTs Sorry for pinging on an IDE-side PR, but want to confirm the correctness from a language perspective. Can both "ClassBlockSyntax" and "ClassStatementSyntax" be considered a "declaration"?